crypto/build.info

```diff
@@ -69,6 +69,7 @@ SOURCE[../providers/fips]=$CORE_COMMON
 $UTIL_COMMON=\
         cryptlib.c params.c params_from_text.c bsearch.c ex_data.c o_str.c \
         ctype.c threads_pthread.c threads_win.c threads_none.c initthread.c \
+        punycode.c \
```

Comment:
Wrong place. The providers aren't using this, so move it to the SOURCE defs of `../libcrypto`

Reply:
Done.